R/lint.R

```diff
@@ -640,16 +640,16 @@ sarif_output <- function(lints, filename = "lintr_results.sarif") {
     as.character(utils::packageVersion("lintr"))
   sarif$runs[[1L]]$originalUriBaseIds$ROOTPATH$uri <- ""
   rule_index_exists <- FALSE
-  root_path_uri <- gsub("\\\\", "/", package_path)
+  root_path_uri <- gsub("\\\\", "/", package_path, fixed = TRUE)
 
   if (startsWith(root_path_uri, "/")) {
-    root_path_uri <- paste("file://", root_path_uri, sep = "")
```

Comment:
wow i'm amazed these survived so long :flushed:

Reply:
Haha, true. This is why I am adding more linters to our lint changed GHA workflow. 
We should be able to catch a few more of such instances in the future.